test/integration/targets/include_import/roles/role1/tasks/fail.yml

```diff
@@ -0,0 +1,2 @@
+- fail:
+    msg: This command should always fail
```

Comment:
The task name must contain `EXPECTED FAILURE` for this to pass CI.

Reply:
Thanks Matt! I'll get that fixed.